doc/rtd/reference/network-config.rst

```diff
@@ -223,6 +223,11 @@ for BSD flavors at the moment.
 Network output policy
 =====================
 
+.. note::
+
+   These are **upstream** defaults and are known to be overridden by
```

Comment:
Do we want to cross link to [cloudinit/net/renderers.py:2](https://github.com/canonical/cloud-init/blob/main/cloudinit/net/renderers.py#L29) here? It may diverge over time, so I'm on the fence about that

Reply:
I don't think so...especially since it's configurable in cloud.cfg